pystac/utils.py

```diff
@@ -11,7 +12,7 @@
 _pathlib = os.path
```

Comment:
This threw me for a loop - I thought it was actual [pathlib](https://docs.python.org/3/library/pathlib.html) for a while.

Reply:
There's an issue around removing this with a suggestion to use mocks for testing. The suggestion would require more test code to implement the patches, but may point to a better way of doing this.